Source/WebCore/platform/graphics/cocoa/VideoMediaSampleRenderer.mm

```diff
@@ -71,12 +72,12 @@ - (void)resetUpcomingSampleBufferPresentationTimeExpectations;
 
 void VideoMediaSampleRenderer::enqueueSample(CMSampleBufferRef sample, bool displaying)
 {
-#if PLATFORM(IOS_FAMILY)
+#if ENABLE(VP9)
     if (!m_decompressionSession && !m_currentCodec) {
         // Only use a decompression session for vp8 or vp9 when software decoded.
         CMVideoFormatDescriptionRef videoFormatDescription = PAL::CMSampleBufferGetFormatDescription(sample);
         auto fourCC = PAL::CMFormatDescriptionGetMediaSubType(videoFormatDescription);
-        if (fourCC == 'vp08' || (fourCC == 'vp09' && !(canLoad_VideoToolbox_VTIsHardwareDecodeSupported() && VTIsHardwareDecodeSupported(kCMVideoCodecType_VP9))))
+        if (fourCC == 'vp08' || (fourCC == 'vp09' && !vp9HardwareDecoderAvailable()))
```

Comment:
Looks ok to me, but please check that it is ok to check the testing override here.

Reply:
we should be consistent across all tests if a decoder is enabled or not.